torch/nn/modules/conv.py

```diff
@@ -32,7 +32,12 @@ def __init__(self, in_channels, out_channels, kernel_size, stride,
         self.transposed = transposed
         self.output_padding = output_padding
         self.groups = groups
+        if padding_mode not in {'zeros', 'reflect', 'replicate', 'circular'}:
+            raise ValueError(
+                "padding_mode must be one of {{'zeros', 'reflect', 'replicate', 'circular'}}, "
```

Comment:
nit: it would be better to assign the set of valid padding modes to a variable so you don't have to repeat it

Reply:
good suggestion. fixed!